backend/app/tests/conftest.py

```diff
@@ -51,6 +52,13 @@ def record_key() -> str:
     return "Nikola Tesla"
 
 
+@pytest.fixture(scope="session")
+def record_type() -> str:
```

Comment:
Keep the stronger type (the `RecordType`) wherever you can rather than loosening it into `str`. The only reason you'd want str here is to pass it to / receive it from the boundaries of your program e.g., to a library or serializing over JSON.